### ISSUE TITLE
Learning Rate for fine tuning the pretrained model

### ISSUE THREAD

**Issue by Pranesh6767** [twtygqyy/pytorch-SRResNet#29]
Hello,
I want to fine-tune/train the pre-trained model for my dataset.... So what learning rate should I use ??
